### PR TITLE
Update dockerfile to use node 12 (lts) base image

### PR DESCRIPTION
**Pull request by dschaper**

### CODE REVIEW

**Comment by dschaper** [comment]
Okay, build is green with new cache version. Ready for review/approval.

File: .circleci/config.yml [hunk 1]
```diff
@@ -26,8 +26,8 @@ jobs:
     - checkout
     - restore_cache:
         keys:
-        - v1-build-{{ checksum "package-lock.json" }}
-        - v1-build-
+        - v2-build-{{ checksum "package-lock.json" }}
+        - v2-build-
```
Comment (XhmikosR) [inline .circleci/config.yml]: This reminds me, need to change this to be foolproof. Don't merge this yet, I will push here a patch tomorrow.

**Comment by XhmikosR** [comment]
This is ready but `codegov` should be installed as a devDependency and not during CI. I'll make this changes after #449.

**Comment by XhmikosR** [comment]
Also do note that with `npm ci` the build will be faster and it will properly fail if the lock file isn't up to date.

**Comment by XhmikosR** [comment]
Rebased and retargeted to development-FTL. Maybe this was meant to go into development, though, so let me know and I will rebase if needed.

**Pull request merged**